### ISSUE TITLE
MacOS: nekoray

### ISSUE THREAD

**Issue by KylixDev** [MatsuriDayo/nekoray#410]
Please update the macOS version of ٔNekoray to the latest version

**Comment by Saulul** [comment]
Yeah I don't know why they stopped providing the Mac packages. Nekoray is probably the only vless/trojan client that works on Mac.

**Comment by y8d4w1** [comment]
我强烈建议恢复MACOS版的放出

**Comment by chromer030** [comment]
For apple-silicon macs : https://github.com/NeetHuang/nekoray-macos-build/releases

For Intel macs : https://github.com/amirhakimnejad/nekoray/releases/

**Comment by Stevemoretz** [comment]
> For apple-silicon macs : https://github.com/NeetHuang/nekoray-macos-build/releases
> 
> For Intel macs : https://github.com/amirhakimnejad/nekoray/releases/

Intel doesn't support 10.15 anymore and required a really high version (13) if you're the owner can you bring the target to 10.15?
